config/settings.py

```diff
@@ -0,0 +1,206 @@
+"""
+Django settings for kitab bazar project.
+
+Generated by 'django-admin startproject' using Django 4.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+import os
+import environ
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+env = environ.Env(
+    DEBUG=(bool, True),
+    SECRET_KEY=(str, 'django-insecure-2s$6zbn2r_0=pao&lf3wz8q%9s*hsi=*)ng=llpkr&3h=#2#85'),
```

Comment:
Let's pass this through docker-compose and not set default. This way developers need to define this on deployment.